Makefile

```diff
@@ -1,5 +1,7 @@
 SHELL := /bin/bash -o pipefail
 
+LAST_SEMVER_TAG := $(shell git tag | sort -r --version-sort | egrep '^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)$$' | head -n1)
```

Comment:
```suggestion
LAST_SEMVER_TAG := $(shell git tag | sort -r --version-sort | egrep '^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)$' | head -n1)
```

Reply:
This is required makefile escaping for dollar signs, i.e. it actually results in a single dollar.